File: app/src/main/java/org/stypox/dicio/settings/HeaderFragment.java
package org.stypox.dicio.settings;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import org.stypox.dicio.R;

public class HeaderFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(final Bundle savedInstanceState, final String rootKey) {
        addPreferencesFromResource(R.xml.pref_header);
    }
}

File: app/src/main/java/org/stypox/dicio/settings/HeaderFragment.kt
package org.stypox.dicio.settings

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import org.stypox.dicio.R

class HeaderFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        addPreferencesFromResource(R.xml.pref_header)
    }
}